jenkins/map.jinja

```diff
@@ -5,9 +5,13 @@
         'netcat_pkg': 'netcat-openbsd',
         'user': 'jenkins',
         'group': 'jenkins',
-        'nginx_user': 'www-data',
-        'nginx_group': 'www-data',
+        'admin_user': 'admin',
+        'admin_pw': 'admin',
+        'nginx_user': 'nginx',
```

Comment:
@nmadhok ok as I am writing this, I just changed the nginx_user and nginx_group back to www-data and in my personal pillar, I am using nginx for both variables. I think we should make this parameterized depending on the os family.